docs/installation/_index.md

```diff
@@ -40,35 +40,25 @@ For an overview of the Viam software platform, see [Learn about Viam](/get-start
 {{% toc %}}
 {{< /alert >}}
 
-## Compatibility
+## Platform requirements
 
 {{< tabs >}}
 {{% tab name="viam-server" %}}
 
-`viam-server` supports:
+`viam-server` can run on any SBC that runs a supported operating system:
```

Comment:
this is not correct bc mac/windows/linux computers do not have to be SBCs-- the board/computer distinction
```suggestion
`viam-server` can run on any computer that runs a supported operating system:
```